apis/v1beta1/vspherefailuredomain_types.go

```diff
@@ -81,12 +82,66 @@ type Topology struct {
 	// +optional
 	Networks []string `json:"networks,omitempty"`
 
+	// NetworkConfigurations is a list with new network configurations within this failure domain
```

Comment:
```suggestion
	// NetworkConfigurations is a list with network configurations within this failure domain.
```

is 'new' redundant here?
add . at the end.

Reply:
"a list `of` network"?